src/phlorest/render.py

```diff
@@ -83,7 +82,7 @@ def render_summary_tree(cldf, output, width=1000):
             if tree.tree_branch_length_unit:
                 legend += ' with branches in {}'.format(tree.tree_branch_length_unit)
             render_tree(
-                tree.newick(),
+                tree.newick(strip_comments=True),
```

Comment:
can we keep comments -- they often contain useful information e.g. https://github.com/phlorest/birchall_et_al2016/blob/9b69ab5ea6bcda8467c90170bf266f107fa4aaf4/cldf/summary.trees#L5

(contains information about rate of change on that branch, min/max/s.d. of length of branch etc etc

Reply:
Comments are only stripped before passing the newick to toytree - because tyotree can't handle them. The dataset still contains the trees + comments.

Reply:
ahh, great